operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -958,6 +963,14 @@ void addStartupEnvVars(List<V1EnvVar> vars, TuningParameters tuningParameters) {
     addEnvVar(vars, ServerEnvVars.SERVICE_NAME, LegalNames.toServerServiceName(getDomainUid(), getServerName()));
     addEnvVar(vars, ServerEnvVars.AS_SERVICE_NAME, LegalNames.toServerServiceName(getDomainUid(), getAsName()));
     Optional.ofNullable(getDataHome()).ifPresent(v -> addEnvVar(vars, ServerEnvVars.DATA_HOME, v));
+    String wdtInstallHome = getWdtInstallHome();
```

Comment:
Is there a reason we need WDT_INSTALL_HOME and WDT_MODEL_HOME environment variables for startServer

Reply:
yes, WDT_INSTALL_HOME is needed in the startServer when decrypting the domain zip file if the WDT install is in a location other than `/u01/wdt/weblogic-deploy`.

Reply:
Oh, ok.  How about WDT_MODEL_HOME - only used in introspection?

Reply:
That env variable will be set only in cases when user sets `wdtModelHome` to a value other than `/u01/wdt/models`. I'm not sure if it's needed for decryption or something else. I can remove it if it's not used.

Reply:
Removed in [af59605](https://github.com/oracle/weblogic-kubernetes-operator/pull/2842/commits/af59605eaa32828fdc1a09c96ae6137ada214924).